posts/2023-02-20-aws-instant-on.adoc

```diff
@@ -0,0 +1,222 @@
+
+NOTE: The cluster nodes are created with the EKS optimized Bottlerocket Amazon Machine Image (AMI) instead of the default Amazon Linux AMI. The Bottlerocket AMI as well as the Ubuntu 20 AMI are using a Linux kernel version that supports InstantOn. As of this writing, the Amazon Linux AMI is running a slightly older kernel version that does not fully support InstantOn.
+
+After making updates to the `cluster.yaml`, run the following command to provision the EKS cluster:
+```
+eksctl create cluster -f cluster.yaml
+```
+
+After 15 or so minutes the cluster should be provisioned and ready to use. Double check by running the following command to see the cluster nodes:
+```
+kubectl get node -o wide
+```
+
+You should see a similar output:
+```
+NAME                             STATUS   ROLES    AGE     VERSION               INTERNAL-IP      EXTERNAL-IP      OS-IMAGE                                KERNEL-VERSION   CONTAINER-RUNTIME
+ip-xxx-xxx-xx-xxx.ec2.internal   Ready    <none>   2m44s   v1.24.6-eks-4360b32   xxx.xxx.xx.xxx   xx.xxx.xxx.xxx   Bottlerocket OS 1.11.1 (aws-k8s-1.24)   5.15.59          containerd://1.6.8+bottlerocket
+ip-xxx-xxx-xx-xxx.ec2.internal   Ready    <none>   2m42s   v1.24.6-eks-4360b32   xxx.xxx.xx.xxx   xx.xx.xx.xx      Bottlerocket OS 1.11.1 (aws-k8s-1.24)   5.15.59          containerd://1.6.8+bottlerocket
+```
+
+=== Deploy InstantOn application
+
+Once the cluster is up and running, we can deploy the Liberty InstantOn application. 
+
+First, create a `deployment.yaml` file with the following contents:
+
+.deployment.yaml
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: open-liberty-instanton
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: open-liberty-instanton
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: open-liberty-instanton
+    spec:
+      containers:
+      - image: <aws_account_id>.dkr.ecr.<region>.amazonaws.com/getting-started-instanton
+        imagePullPolicy: IfNotPresent
+        name: app
+        ports:
+        - containerPort: 9080
+          name: 9080-tcp
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 1
+            memory: 512Mi
+          requests:
+            cpu: 500m
+            memory: 256Mi
+        securityContext:
+          runAsNonRoot: true
+          privileged: false
+          capabilities:
+            add:
+            - CHECKPOINT_RESTORE
+            - SETPCAP
+            drop:
+            - ALL
+----
+
+Make sure to update the `image` value to match the `repositoryUri` from the `create-repository` step. 
+
+Next, deploy the application by running the following:
+```
+kubectl apply -f deployment.yaml
+```
+
+Check the logs to see if the application started up successfully. Keep in mind that it might take a few extra seconds for the pod to start for very first time as the cluster nodes need pull down the container image from ECR:
```

Comment:
```suggestion
Check the logs to see if the application started up successfully. Keep in mind that it might take a few extra seconds for the pod to start for very first time as the cluster nodes must pull down the container image from ECR:
```